### ISSUE TITLE
[Feature]: Find a new name for SimpleX Chat. Suggest potential new names for it.

### ISSUE THREAD

**Issue by ElonSatoshi** [simplex-chat/simplex-chat#4806]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Platform

all

### App version

all

### Feature

The name makes people think of herpes. My friend prefers Briar simply because its name isn't as confusing.

Here are a few horrible suggestions I have:

- **Simple Relay Chat**: It's a slight change to the original name. It's a step away from simplex as in herpes simplex virus. Like the original name, it conveys that the app is simple, and uses relays (SimpleX was a reference to forming simplex one-way connections to users through the relays) Cons: It's longer than the original, it has 3 words instead of 2.
- **Riot**: Element was once called this, maybe we can use it now. It's short, sweet, and simple. Cons: What does the word "riot" have to do at all with this app? Also with Element being previously named Riot, this could generate confusion.

**Comment by ElonSatoshi** [comment]
- **Simple Relay Chat**: It's a slight change to the original name. It's a step away from simplex as in herpes simplex virus. Like the original name, it conveys that the app is simple, and uses relays (SimpleX was a reference to forming simplex one-way connections to users through the relays) Cons: It's longer than the original, it has 3 words instead of 2.

**Comment by ElonSatoshi** [comment]
- **Riot**: Element was once called this, maybe we can use it now. It's short, sweet, and simple. Cons: What does the word "riot" have to do at all with this app? Also with Element being previously named Riot, this could generate confusion.

**Comment by Swamper420** [comment]
Yeah, the name is horrendous. If you search only SimpleX all that comes up is some cryptoscam.

**Comment by droptile** [comment]
This can be solved with some search engine optimization and a bit of marketing. Signal and Telegram also had to overtake the dictionary and wikipedia entries for the words they chose as names in the google search results, and "telegram" and "signal" are much more common words than "simplex." 

I like the name SimpleX/Simplex.

**Comment by ElonSatoshi** [comment]
Search engine optimization won't stop my friend from telling me he likes Briar's name more.

**Comment by jmcelroy01** [comment]
Simplexes are cool - [https://en.wikipedia.org/wiki/Simplex](url)

**Comment by Yiannis128** [comment]
Guys, I think Simple X is a nice name. But is it Simple X or SIMPLEX?

**Comment by Jeffrey2307** [comment]
SimpleX would be the best! :)

**Comment by PrivacyAndSecurity** [comment]
I would agree that just "SimpleX" would be the best as Name, because it is short and everybody already only says SimpleX, instead of SimpleX Chat. The Chat at the End should be taken away and it would be perfect.

Is there any Updates on this?
